strapi/strapi
# Change notification message when we successfully save a created or edited token

What does it do?
When we successfully saved a created or edited token we want to show this Success message

How to test it?
Should satisfy the following criterias:

The confirmation messages are:

 - Edit view: “Success: API Token successfully edited”

 - Creation view: “Success: API Token successfully created”

The duration of the modal with the confirmation messages is the same applied in the rest of the UI
